blog/2024-01-24-embed-figma/index.mdx

```diff
@@ -0,0 +1,75 @@
+
+Two examples of how this can work are:
+
+- Engineers can import state machine code, then designers can fill out images for each state of the actual logic in the code.
+
+- Designers, PMs, and other stakeholders can create the expected behavior by embedding Figma frames into a Stately diagram, then engineers can continue adding the functionality from the automatically scaffolded code.
+
+This seamless integration can speed up the design process and ensure all team members are on the same page. Team members can easily share design details, run critiques, and iterate with people across different specializations.
+
+It also ensures that everyone is always looking at the most up-to-date design, as changes made in Figma are instantly reflected in the embedded view. And with [GitHub Sync](blog/2024-01-09-introducing-bidirectional-github-sync) enabled, changes to the code can even automatically update how the Figma frames are structured in Stately.
+
+## How to embed Figma frames
+
+Figma embedding is part of our [Pro plan](/docs/studio-pro-plan). We offer a trial period if you would like to try it out.
+
+To embed Figma frames in Stately, first you need to authenticate using a personal access token.
+
+1. [Get a personal access token](https://help.figma.com/hc/en-us/articles/8085703771159-Manage-personal-access-tokens) from Figma.
+2. Enter this token in Stately by opening a project, finding the **Stately menu** in the top left, selecting **Account**, opening **Settings**, and clicking the **Figma** tab.
+
+Then you can link frames to states.
+
+3. In Figma, right click on the frame you want to embed, select **Copy/Paste as**, then **Copy link”.
+4. In Stately, select a state, open the **+** menu, choose **Embed Figma**, and paste your link.
+
+[Watch a video of how embedding Figma frames works](https://www.youtube.com/live/c05bOPeinSA?si=qyUlnx7f7_StLDZ_&t=1006) from our recent Office Hours.
+
+![Frame inside Figma showing the menu for copy/paste as, and copy link. Alongside a frame in Stately showing the plus menu and the Embed Figma option.](figmatostately.png)
+
+Now any changes to this frame in Figma will be automatically reflected in Stately after each page load.
+
+You can delete the embed or adjust the size of the image by selecting the state and finding the controls at the top right of the embedded image.
+
+A few more details to get you rolling:
+
+- Stately supports linking to any type of Figma node that you can get,
+  not just frames. But we recommend generally using frames due to how well they
+  correspond to states.
+- Figma embeds are a new type of “asset” in Stately. We also support attaching images, videos, documents, and PDFs to states as assets. You can feature a single asset on the canvas at a time. To feature a different asset, open the state details panel and drag that asset to the first position in the assets list.
```

Comment:
We could link to [Assets in the docs](https://stately.ai/docs/assets), although we could also flesh that page out a bit more with mention of these other formats.